provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving daily cronjob for 1am
```

Comment:
I've made that change here https://github.com/mlandauer/cuttlefish/pull/270/commits/367e9238713cbd5943b141a2738a496433a10fcb

Is the note about AEST too specific to the OAF implementation for upstream?